recipes-graphics/spark/pxcore.inc

```diff
@@ -0,0 +1,18 @@
+SUMMARY = "pxCore is a small open source library that provides a portable framebuffer and windowing abstraction for C++"
+HOMEPAGE = "http://www.sparkui.org/index.html"
+SECTION = "spark"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=e7948fb185616891f6b4b35c09cd6ba5"
+
+DEPENDS_append = " openssl"
+
+PV = "1.7.0+git${SRCPV}"
+
+SRC_URI = "git://github.com/pxscene/pxCore;protocol=git;branch=master"
+SRCREV = "51333037650ecee44191492b541106efa573cc35"
```

Comment:
Can you please add BRANCH variable with weak assignment and make SRCREV as well a weak assignment?

Reply:
done